package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "@graphql-tools/merge": "^9.0.3",
     "@graphql-yoga/plugin-response-cache": "^3.5.0",
+    "@hypercerts-org/contracts": "2.0.0-alpha.0",
```

Comment:
Outdated version

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/pull/177/commits/194cb7330f09d8396b50450b7bfcb8734b104212